onboarding/src/Components/UserEdit/UserEditForm.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useRef, useEffect } from "react";
+import UserProfileManage from "./UserProfileManage";
+import { uploadAvatar, employeeAddEdit } from "../hooks/Users";
+import ModalWarning from "../ModalWarning";
+import { useLocation } from "react-router-dom";
+
+
+function UserEditForm({ user, enableUploadAvatar, buttonTitle, modalTitle, editLoggedUser, setEditLoggedUser }) {
+    const fileNameRef = useRef("");
+    const location = useLocation();
+    const [employeeModal, setModal] = useState(<></>);
+    const [imageLink, updateImageLink] = useState("/onboarding/static/images/unknown-profile.jpg");
+
+    useEffect(() => {
+        updateImageLink(user.avatar || "/onboarding/static/images/unknown-profile.jpg");
```

Comment:
We may need to use the same validation here as in:
https://github.com/binawork/OnlineOnboarding/pull/174/files#r569395358
depending on what is gonna be in `App.js` on line around 70:
`<EmployeeEditPage />`